jitsi-media-transform/src/test/kotlin/org/jitsi/nlj/codec/vpx/PictureIdIndexTrackerTest.kt

```diff
@@ -20,6 +20,7 @@ import io.kotest.core.spec.IsolationMode
 import io.kotest.core.spec.style.ShouldSpec
 import io.kotest.matchers.comparables.shouldBeGreaterThan
 import io.kotest.matchers.shouldBe
+import org.jitsi.nlj.util.PictureIdIndexTracker
 
 internal class PictureIdIndexTrackerTest : ShouldSpec() {
```

Comment:
This test should probably get moved to the namespace that PictureIdIndexTracker is in now.